### PR TITLE
Replace tickerinterval

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
replace all internal occurences of `ticker_interval` with timeframe.
This does not yet touch the user interface (i'd like to do this in a seperate step as this can be painfull if not done correctly).

Doing this first keeps the PR concis(er) and should make reviewing easier (there are still over 100 "ticker_interval" left in the code ... all "user-visible".

Part of issue: #2318

## Quick changelog

- replace all internal occurences of `ticker_interval` with timeframe.

### CODE REVIEW

File: freqtrade/data/history.py [hunk 1]
```diff
@@ -276,18 +276,18 @@ def _load_cached_data_for_updating(datadir: Path, pair: str, ticker_interval: st
 def download_pair_history(datadir: Path,
                           exchange: Optional[Exchange],
                           pair: str,
-                          ticker_interval: str = '5m',
+                          timeframe: str = '5m',
                           timerange: Optional[TimeRange] = None) -> bool:
     """
     Download the latest ticker intervals from the exchange for the pair passed in parameters
```
Comment (hroff-1902) [inline freqtrade/data/history.py]: "Download the latest timeframes from the exchange"

This maybe should better be rephrased (like "Download latest candles..." or "Download data for latest timeframes...")

**Comment by hroff-1902** [comment]
Some more changes requested ^^, plus:
* parameter for `timeframes_*(ticker_interval, ...)` in exchange/misc.py

**Comment by xmatthias** [comment]
> parameter for timeframes_*(ticker_interval, ...) in exchange/misc.py

not sure where you mean .. ? the timeframes_* methods in the exchange class do have `timeframe` already ... ?

**Comment by hroff-1902** [comment]
> > parameter for timeframes_*(ticker_interval, ...) in exchange/misc.py
> 
> not sure where you mean .. ? the timeframes_* methods in the exchange class do have `timeframe` already ... ?

Ah, it was in my local file that left aside (did not go to the stashes since not staged)
I split those functions into a separate exchange.misc module...
So forget this.
I need to cleanup my local files after that feat/new-args merged...

**Pull request merged**